[BAEL-4635] Tests for JPA Transient Annotation tutorial

thejoeflow opened the pull request.

JonCook reviewed (changes requested):
Hi @thejoeflow 

Just a few things to fix here:

- Let's avoid using e.printStackTrace(); - let's either use a proper logger or remove them
- Unit test class names need to end in UnitTest, integration tests with IntegrationTest, etc.
- Follow our unit test naming convention (givenX_whenY_thenZ) as per the author guidelines.

Thanks

thejoeflow commented:
Hey Jon, Thanks for the feedback. Still getting used to the coding guidelines and conventions.

-     Removed all usage of e.printStackTrace().
-     Renamed test class TransientAnnotationUnitTest
-     Renamed unit tests using standard givenX_whenY_thenZ notation

Let me know if there's anything else!
